scripts/pin_the_slow_way.py

```diff
@@ -32,6 +32,7 @@
           'fontconfig': 'fontconfig 2.11.*',
           'freetype': 'freetype 2.6.*',
           'geos': 'geos 3.4.*',
+          'giflib': 'giflib 5.1.2|5.1.2.*',
```

Comment:
My recommendation is a hard pin on 5.1.2 but people here are usually against it.